new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleDomainValidation.java

```diff
@@ -48,6 +55,51 @@ public void testCreatingDomain() {
             .metadata(new V1ObjectMeta().namespace(namespace).name(serviceAccountName))));
     logger.info("Created service account: {0}", serviceAccount.getMetadata().getName());
 
+    // create persistent volume and persistent volume claim
+    final String pvcName = domainUID + "-pvc"; // name of the persistent volume claim
+    final String pvName = domainUID + "-pv"; // name of the persistent volume
+
+    V1PersistentVolumeClaim v1pvc = new V1PersistentVolumeClaim()
+        .spec(new V1PersistentVolumeClaimSpec()
+            .addAccessModesItem("ReadWriteMany")
+            .storageClassName(domainUID + "-weblogic-domain-storage-class")
+            .volumeName(domainUID + "-weblogic-pv")
+            .resources(new V1ResourceRequirements()
+                .putRequestsItem("storage", Quantity.fromString("10Gi"))))
+        .metadata(new V1ObjectMetaBuilder()
+            .withName(pvcName)
+            .withNamespace(namespace)
+            .build()
+            .putLabelsItem("weblogic.resourceVersion", "domain-v2")
+            .putLabelsItem("weblogic.domainUID", domainUID));
+
+    boolean success = assertDoesNotThrow(
+        () -> TestActions.createPersistentVolumeClaim(v1pvc),
+        "Persistent volume claim creation failed, look at the log for failure ApiException responsebody"
```

Comment:
it would be better to catch the exception and check it, and what is "look at the log"? what log? where? get the details and print them out here yourself

Reply:
The final call to create pvc/pv "Kubernetes.createPersistentVolumeClaim(v1pvc)" will log ApiException messages as severe in the console. That is addressed in PR https://github.com/oracle/weblogic-kubernetes-operator/pull/1562

When this messages is printed
"Persistent volume claim creation failed, look at the log for failure ApiException responsebody"

The reason for the failure will be just above few lines in the console log.

So which way you want to check ?

assertDoesNotThrow(
        () -> TestActions.createPersistentVolumeClaim(v1pvc),
        "Persistent volume claim creation failed, "
            + "look at the above log messages for failure reason in ApiException responsebody"); 

(or)

try{
      TestActions.createPersistentVolumeClaim(v1pvc);
      logger.info("PersistentVolumeClaim {0} created", pvcName);
      } catch (ApiException ex) {
      logger.severe("PersistentVolumeClaim {0} creation failed.", pvcName);
      logger.severe(ex.getResponseBody());
      fail(String.format("PersistentVolumeClaim {0} creation failed", pvcName));
    }

Reply:
This is how it will fail.

<04-15-2020 18:27:03> <INFO> <oracle.weblogic.kubernetes.ItSimpleDomainValidation testCreatingDomain> <Created service account: ns-nrko-sa>
<04-15-2020 18:27:03> <SEVERE> <oracle.weblogic.kubernetes.actions.impl.primitive.Kubernetes createPvc> <{"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"persistentvolumeclaims \"domain1-pvc\" already exists","reason":"AlreadyExists","details":{"name":"domain1-pvc","kind":"persistentvolumeclaims"},"code":409}
>
<04-15-2020 18:27:03> <INFO> <oracle.weblogic.kubernetes.extensions.Timing afterTestExecution> <Method [testCreatingDomain] took 978 ms.>
<04-15-2020 18:27:03> <INFO> <oracle.weblogic.kubernetes.extensions.LoggedTest afterEachTest> <Finished executing [Create a domain] in oracle.weblogic.kubernetes.ItSimpleDomainValidation.testCreatingDomain()>
[ERROR] Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 1.026 s <<< FAILURE! - in oracle.weblogic.kubernetes.ItSimpleDomainValidation
[ERROR] testCreatingDomain  Time elapsed: 1.02 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: Persistent volume claim creation failed, look at the above log messages for failure reason in ApiException responsebody ==> Unexpected exception thrown: io.kubernetes.client.openapi.ApiException
        at org.junit.jupiter.api.AssertDoesNotThrow.createAssertionFailedError(AssertDoesNotThrow.java:83)
        at org.junit.jupiter.api.AssertDoesNotThrow.assertDoesNotThrow(AssertDoesNotThrow.java:76)
        at org.junit.jupiter.api.AssertDoesNotThrow.assertDoesNotThrow(AssertDoesNotThrow.java:63)
        at org.junit.jupiter.api.Assertions.assertDoesNotThrow(Assertions.java:3086)
        at oracle.weblogic.kubernetes.ItSimpleDomainValidation.testCreatingDomain(ItSimpleDomainValidation.java:97)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)